src/spikeinterface/sorters/basesorter.py

```diff
@@ -21,11 +21,11 @@
 from .utils import SpikeSortingError, ShellScript
 
 
-default_job_kwargs = {"n_jobs": -1}
+default_job_kwargs = {}
```

Comment:
We can. Once we decide on the global I get do both changes in a new commit.

Reply:
Cool. I'm in the recording suite for the next couple days. If you are trying to push this with a deadline definitely just close this (or take over this one), otherwise I will do that in a couple days/if one of my experiments ends early.